redaxo/src/addons/phpmailer/README.md

```diff
@@ -173,5 +173,5 @@ $mail->SMTPOptions = array(
 <a name="multipledomains"></a>
 ### Senden über multiple Domains 
 
-Liedert ein AddOn oder Modul keine eigenen Sende-Einstellungen für das Versenden der Mails, gilt:
+Liefert ein AddOn oder Modul keine eigenen Sende-Einstellungen für das Versenden der Mails, gilt:
```

Comment:
`Sende-` würde ich löschen und `Mails` in `E-Mails`ändern

Reply:
danke